### PR TITLE
Issue:3903841:PDR plugin: adding warnings and protection

### PR DESCRIPTION
**Pull request by vg12345**
[#3903841](https://redmine.mellanox.com/issues/3903841)

### CODE REVIEW

**Comment by vg12345** [comment]
https://redmine.mellanox.com/issues/3903841

File: plugins/pdr_deterministic_plugin/ufm_sim_web_service/api/pdr_plugin_api.py [hunk 1]
```diff
@@ -116,3 +121,27 @@ def get_request_data(self):
...
+
+        # Additional corrections can be added here upon request
+        return port_name
+
+    def get_port_warning(self, port_name):
+        """
+        Return warning text if port does not exist in ports data and update plugin logs
+        """
+        if not self.isolation_mgr.test_mode:
+            if self.isolation_mgr.ports_data.get(port_name) is None:
```
Comment (drorlevy) [inline plugins/pdr_deterministic_plugin/ufm_sim_web_service/api/pdr_plugin_api.py:143]: if not ... instead of "is None"

Comment (ananalaghbar) [reply plugins/pdr_deterministic_plugin/ufm_sim_web_service/api/pdr_plugin_api.py:143]: +1

**Review by ananalaghbar — APPROVED** [review]

File: plugins/pdr_deterministic_plugin/ufm_sim_web_service/api/pdr_plugin_api.py (hunk 1 above)
Comment (vg12345) [reply plugins/pdr_deterministic_plugin/ufm_sim_web_service/api/pdr_plugin_api.py]: Fixed

**Review by drorlevy — APPROVED** [review]

**Pull request merged**